src/CGIHandler.hpp

```diff
@@ -3,21 +3,25 @@
 
 #include "ARequestHandler.hpp"
 #include "HTTPRequest.hpp"
-#include "HTTPResponse.hpp"
 #include "Environment.hpp"
+#include <unistd.h>
+#include <sys/wait.h>
+#include <fcntl.h>
+#include <vector>
 
 class CGIHandler : public ARequestHandler
 {
   public:
 	CGIHandler();
 	virtual ~CGIHandler();
-	virtual HTTPResponse handleRequest(const HTTPRequest &request);
-	// std::string executeCGI(const HTTPRequest &request, const Environment &env);
-	std::string executeCGI(const char *argv[], const Environment &env);
+	HTTPResponse handleRequest(const HTTPRequest &request);
```

Comment:
Check if we really needs three methods to handle and execute the CGI request.

Reply:
refactored